tests/relay/test_nome_download.py

```
@@ -0,0 +1,160 @@
+
+def test_il_profilo_nominato_scarica_col_suo_nome(tmp_path, monkeypatch):
+    relay_in_processo(monkeypatch, tmp_path / 'profilo.db')
+    r = main.named_profile_csv(main.PIERO_PROFILE, token=TOKEN_DI_PROVA)
+    assert r.headers.get('content-disposition') == \
+        f'attachment; filename="betrelay-{main.PIERO_PROFILE}.csv"'
+
+
+def test_il_feed_utente_scarica_con_lo_slug(tmp_path, monkeypatch):
+    percorso = relay_in_processo(monkeypatch, tmp_path / 'feed.db')
+    _utente_con_feed(percorso)  # slug='marco'
+    r = main.feed_utente_csv('marco', token='xt_token-di-prova-abcdef')
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "token"

(S106)